spark-statsd-reporter/src/main/java/org/apache/spark/metrics/sink/statsd/InstanceDetailsProvider.java

```diff
@@ -38,6 +38,7 @@ InstanceDetails buildInstanceDetails() {
         return new InstanceDetails(
                 sparkConf.getAppId(),
                 sparkConf.get("spark.app.name"),
+                System.getenv("SPARK_APP_ORIGIN"),
                 InstanceType.valueOf(SparkEnv.get().metricsSystem().instance().toUpperCase()),
```

Comment:
let's switch to `System.getEnv("SPARK_INSTANCE_TYPE")` this will allow us to move away from `SparkEnv` in an easier way later on.

Reply:
changed as suggested